control/Controller.php

```diff
@@ -69,6 +69,13 @@ class Controller extends RequestHandler implements TemplateGlobalProvider {
 	 * @uses BasicAuth::requireLogin()
 	 */
 	public function init() {
```

Comment:
But that is slow and nasty :(

I don't think we should start filling up __call with all our deprecation notices...